### DISCUSSION TITLE
Dialogic in Godot Mobile?

### DISCUSSION THREAD

**Discussion by DevMenor** [dialogic-godot/dialogic#2075] [General]
I want to know if the dialogic plugin is already available for godot 4.2.1 mobile, as I have already installed it several times and it keeps giving me an error.
....res://dialogic-main/Tests/Unit/test_example.gd:2 - Parse Error: Could not find base class "GdUnitTestSuite".
  modules/gdscript/gdscript.cpp:2788 - Failed to load script "res://dialogic-main/Tests/Unit/test_example.gd" with error "Parse error". (User)....

**Comment by CakeVR** [comment]
Hello, your error has nothing to do with Android but you didn't follow the installation steps.
Take a look here: https://dialogic-docs.coppolaemilio.com/getting-started.html#1-installation--activation
